### DISCUSSION TITLE
AEGIS in kryptor

### DISCUSSION THREAD

**Discussion by sidielnikov** [samuel-lucas6/Kryptor#61] [General]
Hi saw your AEGIS.NET repositories. Will ChaCha20 be replaced by a cryptor for AEGIS and when?

**Comment by samuel-lucas6** [comment]
I haven't decided yet but quite possibly. If so, it won't be happening for ages:

1. It's a breaking change, and I'd group multiple breaking changes together, meaning new file formats and other changes too. None of that is trivial and requires a lot of time and care. I don't have the time currently and want to eventually reach a format/protocol I'm (nearly) completely happy with.
2. I'd use the implementation in libsodium, which means waiting for the next release.
